tests/features/steps/client.py

```diff
@@ -50,4 +50,13 @@ def step_impl(context):
     'http': 'http://10.10.1.10:3128',
     'https': 'http://10.10.1.10:1080',
     }
-    assert context.soccer.SESSION.proxies == proxies
\ No newline at end of file
+    assert context.soccer.SESSION.proxies == proxies
+
+@when(u'the cache is cleared')
+def step_impl(context):
+    context.soccer.clear_cache()
+
+@then(u'the cache is empty')
+def step_impl(context):
+    for adapter in context.soccer.session.adapters:
+        assert adapter.cache.data == {}
```

Comment:
Looks like you've got a test failing here.

Reply:
Yeah, I thought this would work based on the comment here: https://github.com/psf/cachecontrol/issues/237#issuecomment-867644032

But I might need to try something else

Reply:
Gave this a go: https://cachecontrol.readthedocs.io/en/latest/custom_heuristics.html#expires-after